app/src/main/java/com/epfl/drawyourpath/database/FireDatabase.kt

```diff
@@ -10,6 +10,17 @@ private val TIMEOUT_SERVER_REQUEST: Long = 10
 
 class FireDatabase : Database() {
     val database: DatabaseReference = Firebase.database.reference
+    override fun isUserStoreOnDatabase(userId: String): CompletableFuture<Boolean> {
```

Comment:
Should probably fix `private fun updateUserData(data: Map<String, Any>, username: String)` to use the `uid` instead and use this function here.

Reply:
I will do that in my next task(rebase the database). I have added a TODO comment to think about it. Thanks